eng/pipelines/runtimelab/install-llvm.ps1

```diff
@@ -35,23 +41,45 @@ foreach ($Config in $Configs)
 {
     pushd llvm-project
     $BuildDirName = "build-$($Config.ToLower())"
-    mkdir $BuildDirName -Force
+    if ($IsWindows)
+    {
+        mkdir $BuildDirName -Force
+    }
+    else
+    {
+        mkdir $BuildDirName --parents
+    }
```

Comment:
```suggestion
    New-Item $BuildDirName -Force
```
`mkdir` is a PS alias; not using should avoid the need for the condition.

Reply:
Thanks, done with `-ItemType Directory`